src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -81,23 +83,35 @@ public override ExportResult Export(in Batch<LogRecord> batch)
                 this.WriteLine($"{"LogRecord.Body:",-RightPaddingLength}{logRecord.Body}");
             }
 
-            if (logRecord.Attributes != null)
+            if (logRecord.Attributes != null || logRecord.Exception != null)
             {
                 this.WriteLine("LogRecord.Attributes (Key:Value):");
-                for (int i = 0; i < logRecord.Attributes.Count; i++)
+                if (logRecord.Attributes != null)
                 {
-                    // Special casing {OriginalFormat}
-                    // See https://github.com/open-telemetry/opentelemetry-dotnet/pull/3182
-                    // for explanation.
-                    var valueToTransform = logRecord.Attributes[i].Key.Equals("{OriginalFormat}")
-                        ? new KeyValuePair<string, object>("OriginalFormat (a.k.a Body)", logRecord.Attributes[i].Value)
-                        : logRecord.Attributes[i];
-
-                    if (ConsoleTagTransformer.Instance.TryTransformTag(valueToTransform, out var result))
+                    for (int i = 0; i < logRecord.Attributes.Count; i++)
                     {
-                        this.WriteLine($"{string.Empty,-4}{result}");
+                        // Special casing {OriginalFormat}
+                        // See https://github.com/open-telemetry/opentelemetry-dotnet/pull/3182
+                        // for explanation.
+                        var valueToTransform = logRecord.Attributes[i].Key.Equals("{OriginalFormat}")
+                            ? new KeyValuePair<string, object>(
+                                "OriginalFormat (a.k.a Body)",
+                                logRecord.Attributes[i].Value)
+                            : logRecord.Attributes[i];
+
+                        if (ConsoleTagTransformer.Instance.TryTransformTag(valueToTransform, out var result))
+                        {
+                            this.WriteLine($"{string.Empty,-4}{result}");
+                        }
                     }
                 }
+
+                if (logRecord.Exception != null)
```

Comment:
Move this `if` block outside of the `if (logRecord.Attributes != null)` check.

Reply:
Same reasoning as above: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5351#discussion_r1494981092